test-data/src/test/resources/org/openmrs/module/fhir2/providers/PatientWebTest_create.json

```diff
@@ -1,5 +1,6 @@
 {
   "resourceType": "Patient",
+  "active": true,
```

Comment:
The "active" attribute should no longer be translated, so I don't quite understand why we need it here?

Reply:
We need it because in the patient translator we are checking if the patient instance is active or not, and if this active field is not present, it is throwing an exception saying `voided by fhir module`

Reply:
The easier solution is just to remove that check.